docker/spec/dependabot/docker/version_spec.rb

```diff
@@ -27,13 +27,19 @@
   end
 
   describe ".correct?" do
+    def check_version_for_correctness?(version)
+      docker_version = described_class.new(version)
+      described_class.correct?(docker_version)
+    end
+
     it "classifies standard versions as correct" do
-      expect(described_class.correct?("2.4.2")).to be true
+      expect(check_version_for_correctness?("2.4.2")).to be true
     end
 
     it "classifies java versions as correct" do
-      expect(described_class.correct?("11.0.16_8")).to be true
-      expect(described_class.correct?("11.0.16.1")).to be true
+      expect(check_version_for_correctness?("11.0.16_8")).to be true
+      expect(check_version_for_correctness?("v11.0.16_8")).to be true
+      expect(check_version_for_correctness?("11.0.16.1")).to be true
```

Comment:
I think this should work and reads a bit nicer:

```suggestion
      expect(described_class.new("11.0.16_8")).to be_correct
      expect(described_class.new("v11.0.16_8")).to be_correct
      expect(described_class.new("11.0.16.1")).to be_correct
```

This works because of https://www.rubydoc.info/gems/rspec-expectations/RSpec%2FMatchers:be